metrics/metrics.go

```diff
@@ -111,14 +108,34 @@ func parseMetricsFile(path string) (*File, error) {
 	metrics := make(map[string]string)
 
 	for k, v := range tmpMetrics {
-		s, err := json.Marshal(v)
-		if err != nil {
-			l.Errorw("error during marshalling metric value to JSON, skipping",
-				zap.Any("value", v),
-				zap.Error(err))
-			return nil, err
+		switch reflect.TypeOf(v).Kind() { //nolint:exhaustive
+		case reflect.String:
+			if vs, ok := v.(string); ok {
```

Comment:
Does it make sense to check `ok` in type assertion if you already checked type with reflection? Also you can do the same without reflection at all: https://github.com/percona/platform/blob/55498ecf2d969de75ce38cdc1f234d9fbdf5f620/pkg/starlark/values.go#L20

Reply:
done